.ci/make.sh

```diff
@@ -164,7 +164,13 @@ if [[ "$CMD" == "bump" ]]; then
 fi
 
 if [[ "$CMD" == "codegen" ]]; then
-    echo "TODO"
+  docker run \
```

Comment:
I think some extra changes may have leaked into this branch?

Reply:
Yes, it was a mistake!